Assets/Scripts/ApplicationLifecycle/Unity.BossRoom.ApplicationLifecycle.asmdef

```diff
@@ -9,7 +9,8 @@
         "GUID:1491147abca9d7d4bb7105af628b223e",
         "GUID:e0cd26848372d4e5c891c569017e11f1",
         "GUID:e9e5562f77c754b2ca364915385b3d43",
-        "GUID:59b3bd604d0445f583783872c3e648fc"
+        "GUID:59b3bd604d0445f583783872c3e648fc",
```

Comment:
it'd be good if we came back to full text references now that the moves have been done, this is hard to review.
What's the added ref?

Reply:
It's VContainer, and yeah - we can revert to full-text no probs.